### PR TITLE
Add tx pending alert when tx interacting with the cowamm

### PR DESCRIPTION
**Pull request by yvesfracari**
image

### CODE REVIEW

File: apps/cow-amm-deployer/src/components/TxPendingAlertCard.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { AlertCard } from "./AlertCard";
+
+export function TxPendingAlertCard() {
+  return (
+    <div className="bg-foreground text-background mb-2">
+      <AlertCard style="warning" title="CoW AMM Transaction pending">
+        <p>
+          There is a pending transaction related to the CoW AMM. Please take
+          care staking multiple CoW AMM actions since they can interfere with
```
Comment (ribeirojose) [inline apps/cow-amm-deployer/src/components/TxPendingAlertCard.tsx:9]: wdym by "staking" here? 🤔

File: apps/cow-amm-deployer/src/lib/fetchHasAmmTxPending.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      | MultiSend
+    )[];
+
+    const anyTxToProductConstantFactory = txInfos.some(
+      (txInfo) =>
+        txInfo.to.value.toLowerCase() ===
+        COW_CONSTANT_PRODUCT_FACTORY[chainId].toLowerCase()
+    );
+
+    // early return to avoid unnecessary calls to getTransactionDetails
```
Comment (ribeirojose) [inline apps/cow-amm-deployer/src/lib/fetchHasAmmTxPending.ts:35]: 👍

File: apps/cow-amm-deployer/src/lib/fetchHasAmmTxPending.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        if (!txsInsideMultisend) return false;
+        return txsInsideMultisend.some(
+          (tx) =>
+            tx.to.toLowerCase() ===
+            COW_CONSTANT_PRODUCT_FACTORY[chainId].toLowerCase()
+        );
+      }
+    );
+
+    return anyMultiSendTxToProductConstantFactory;
```
Comment (ribeirojose) [inline apps/cow-amm-deployer/src/lib/fetchHasAmmTxPending.ts:65]: 👌

**Review by ribeirojose — APPROVED** [review]

File: apps/cow-amm-deployer/src/components/TxPendingAlertCard.tsx (hunk 1 above)
Comment (yvesfracari) [reply apps/cow-amm-deployer/src/components/TxPendingAlertCard.tsx:9]: Like adding multiple transactions to a queue.

Comment (ribeirojose) [reply apps/cow-amm-deployer/src/components/TxPendingAlertCard.tsx:9]: so `stacking`?

**Pull request merged**